### DISCUSSION TITLE
interesting paper of current research on Learning Objects in Virtual Learning Environments

### DISCUSSION THREAD

**Discussion by abotzki** [LiaScript/LiaScript#152] [General]
@andre-dietrich You might have seen it when you were in Aachen where you presented Liascript as a 1/2 day workshop at that conference in autumn last year.

Fine-GrainedLearning Objects in Virtual Learning Environments
https://dl.gi.de/server/api/core/bitstreams/2edac66b-5713-4795-a467-7e6c75ea6727/content

A lot of challenges lie in front of us...

**Comment by andre-dietrich** [comment]
Thanks for sharing ;-) I just read it. Unfortunately I did not attend to a lot of presentations, we also had a hands-on demo the other day (not running at first ;-)) ...

https://dl.gi.de/server/api/core/bitstreams/91eed94d-43a6-4de0-b6a9-7472bab58a6d/content 

I could not find the implementation, but I think Fine-GrainedLearning Objects could be used in two directions ... On the one side they could be used directly in LiaScript to integrate additional resources dynamically, videos, texts, quizzes ... That should be possible already by using scripts ... I simply created a little example on how to query OERSI with LiaScript, but the data still needs to be turned into something more relevant than Json ... 

![Bildschirmfoto vom 2024-01-12 12-56-13](https://github.com/LiaScript/LiaScript/assets/3089101/f1a7184f-03bb-4061-a599-905641d23015)

Here is the executable example in the [LiveEditor](https://liascript.github.io/LiveEditor/?/show/code/H4sIAAAAAAAAA6VUXW/TMBR9z6+4Mg9JRJsMhHhI1yJAkxhfQ2xvE5rc+LYxc+xgO+vC1v/OdbJ2bdkDEs5D4utzfH3Ovc4zODv5fn46PkduyyqKjqVuWg8R0PBdg1Pm8dazfu58pygwN1agLeBFcwvOKClgrnh5PYEK5bLyBbyyWE+g5nYpNcGOCHcUnsmwjRRT5vp0w/yGq5a2/aRkzVdcC1QsmkXH89Z7o3uE0aWS5fWGlqT/e56QYFA8yI+O8yHdjBxwpZWNB9vqMSVG6B2hND2AQW2EXHRTtuDKIdscQkjXKN4VlNuE3DW/HW/yvz66qSZgbtAulFkVwFtv2CxaSS3MKhs0wRQWrS69NDpJ4a7XVxrtPGzXhSnbGrXPluhPFIbPd92pSDZmplnvJNzfAxsM+lfGNBB6xgI9+RtX3jeuyHOD1snM2GVu0ZnWluhy3sh84IflK8E9z6+GwJvGovddPHoQEEaNvjKigPjb2flFPNrGK+RUNVfsQMOI3xvt6ZzjC2q+mGi8aaj2PBiT/3RG72zR49+WJTb+SeQWuH7kzI2gIn08P/uaOW+lXlItk/0jOPkbC3h5tJ9oYU1dwEHwV4u2O5TQq26Vl1c1JzefWt7hDs6NnoQsJCpBFl0yzWtkI2ACh+4kjWF6jd2Kut+xH3/x13uR9X4CZyz15eUd3cUCGHclg/XjFus02nllvkKdUP0bakfqlRlsvrPgcpLuwkI3BMij5DwnQ+tGdRD+KwQKdHKHrpU3wDV8uPjyeVwagVuKQy0yJXnCwloBx2H14aLFq0p6HLuGl1Qk6rfxyvImnjF4vqfwoMLhXCPQrVIjeJkeYFmfYMa2wX39ZahikqC1xqZBXLiXRmHWR5L4JLwKavoBkU6idbSBKLPc3DYSLL3kirpLsJT+OEMlZ38AGS+6kIIFAAA=)

On the other hand this provides an interesting opportunity for (re)-combining resources into a single LiaScript document, I mean generating a course automatically ... maybe also with the help of AI as described here:

https://aneesha.medium.com/act-as-a-learning-designer-getting-chatgpt-to-generate-an-online-module-8a16a2813bd6

**Comment by abotzki** [comment]
@andre-dietrich Thanks for sharing the link to OERSI - very nice platform. I will show this to my colleagues at the institute and also in our network. 
Yes, I also think so that these are two use cases. Potentially, there are other options.

The paper seems to go into yet another direction, use ML/AI for proposing relevant content to the learner if we were to understand what the persons needs in the learning process.

I think that [here](https://iug-research.gitlab.io/oer-conditional-probability/) is a demo of their approach, not sure whether the ML/AI part has been implemented yet, though. There are some nice interactive examples in this demo course. e.g. https://iug-research.gitlab.io/oer-conditional-probability/#/5
